chromium_src/components/history/core/browser/DEPS

```diff
@@ -0,0 +1,7 @@
+# Existing exceptions
+specific_include_rules = {
+  "history_backend.cc": [
```

Comment:
when you had to touch `chromium_src/**/DEPS` usually means you also need to do this for gn files.

please make sure `//components/history/core/browser` has the right dependency on `//brave/components/brave_sync`.

to test just add `#include "brave/components/brave_sync/features.h"` directly into upstream file `components/history/core/browser/history_backend.cc` and run gn check.

Reply:
And just in case error at `npm run gn_check` with 

>add #include "brave/components/brave_sync/features.h" directly into upstream file components/history/core/browser/history_backend.cc and run gn check.

```
> gn check /home/alexey/Projects/br_br_sync_history_soner/brave-browser/src/out/Component //components/*
ERROR at //components/history/core/browser/history_backend.cc:6:15: Can't include this header from here.
    #include "brave/components/brave_sync/features.h"
              ^-------------------------------------
The target:
  //components/history/core/browser:browser
is including a file from the target:
  //brave/components/brave_sync:features

It's usually best to depend directly on the destination target.
In some cases, the destination target is considered a subcomponent
of an intermediate target. In this case, the intermediate target
should depend publicly on the destination to forward the ability
to include headers.

Dependency chain (there may also be others):
  //components/history/core/browser:browser -->
  //components/sync:sync -->
  //components/sync/driver:driver --[private]-->
  //brave/components/brave_sync:brave_sync --[private]-->
  //brave/components/brave_sync:features
  ```

Reply:
So I could add a public deps 
`//components/sync:sync -->  //brave/components/brave_sync:features`

But `src/components/sync/BUILD.gn` doesn't have any Brave's patches, so I decided patch `src/components/history/core/browser/BUILD.gn` as the most closest place to one where I need my new deps.